packages/st2/debian/rules

```diff
@@ -37,6 +37,7 @@ ifeq (true, $(IS_SYSTEMD))
 	dh_systemd_enable --name=st2rulesengine st2rulesengine.service
 	dh_systemd_enable --name=st2sensorcontainer st2sensorcontainer.service
 	dh_systemd_enable --name=st2garbagecollector st2garbagecollector.service
+    dh_systemd_enable --name=st2workflowengine st2workflowengine.service
```

Comment:
Use `tabs` instead of spaces in `Makefile` for indentation as that's the only supported way by their syntax.